sdk/samples/iot/CMakeLists.txt

```diff
@@ -67,39 +67,30 @@ target_link_libraries(paho_iot_hub_methods_sample
 
 create_map_file(paho_iot_hub_methods_sample paho_iot_hub_methods_sample.map)
 
-# PnP Sample
-add_executable (paho_iot_hub_pnp_sample
-  ${CMAKE_CURRENT_LIST_DIR}/paho_iot_hub_pnp_sample.c
+add_executable (paho_iot_pnp_sample
+  ${CMAKE_CURRENT_LIST_DIR}/paho_iot_pnp_sample.c
+  ${CMAKE_CURRENT_LIST_DIR}/paho_iot_pnp_sample_common.c
 )
 
-target_link_libraries(paho_iot_hub_pnp_sample
+target_link_libraries(paho_iot_pnp_sample
   PRIVATE
     az::iot::sample::common
 )
 
-create_map_file(paho_iot_hub_pnp_sample paho_iot_hub_pnp_sample.map)
+create_map_file(paho_iot_pnp_sample paho_iot_pnp_sample.map)
 
-# PnP Component Sample
-add_executable (paho_iot_hub_pnp_component_sample
-  ${CMAKE_CURRENT_LIST_DIR}/pnp/pnp_mqtt_message.c
-  ${CMAKE_CURRENT_LIST_DIR}/pnp/pnp_device_info_component.c
-  ${CMAKE_CURRENT_LIST_DIR}/pnp/pnp_protocol.c
-  ${CMAKE_CURRENT_LIST_DIR}/pnp/pnp_thermostat_component.c
-  ${CMAKE_CURRENT_LIST_DIR}/paho_iot_hub_pnp_component_sample.c
+# PnP Sample
```

Comment:
```suggestion
# PnP with Provisioning Sample
```

Reply:
Nit:  Can you move the PnP sample cmake code to after the Twin Sample and before Provisioning Samples?  Then it will be in the same order as presented in the Readme

Reply:
Fixed.